### PR TITLE
Implement `conversations` API

### PR DESCRIPTION
**Pull request by joeltoby**
Implemented the [`conversations` API](https://api.slack.com/docs/conversations-api) along with all 17 methods.

Resolves issue #37.

### CODE REVIEW

File: src/main/java/com/github/seratch/jslack/api/methods/impl/MethodsClientImpl.java [hunk 1]
```diff
@@ -344,6 +504,191 @@ public ChatUpdateResponse chatUpdate(ChatUpdateRequest req) throws IOException,
     }
     
     @Override
+    public ConversationsArchiveResponse conversationsArchive(ConversationsArchiveRequest req)
+	    throws IOException, SlackApiException {
+	FormBody.Builder form = new FormBody.Builder();
+        setIfNotNull("token", req.getToken(), form);
```
Comment (seratch) [inline src/main/java/com/github/seratch/jslack/api/methods/impl/MethodsClientImpl.java]: Please adjust indents.

File: src/main/java/com/github/seratch/jslack/api/methods/impl/MethodsClientImpl.java [hunk 2]
```diff
@@ -1,27 +1,96 @@
...
+
 import com.github.seratch.jslack.api.methods.Methods;
 import com.github.seratch.jslack.api.methods.MethodsClient;
 import com.github.seratch.jslack.api.methods.SlackApiException;
 import com.github.seratch.jslack.api.methods.request.api.ApiTestRequest;
 import com.github.seratch.jslack.api.methods.request.auth.AuthRevokeRequest;
 import com.github.seratch.jslack.api.methods.request.auth.AuthTestRequest;
 import com.github.seratch.jslack.api.methods.request.bots.BotsInfoRequest;
-import com.github.seratch.jslack.api.methods.request.channels.*;
+import com.github.seratch.jslack.api.methods.request.channels.ChannelsArchiveRequest;
```
Comment (seratch) [inline src/main/java/com/github/seratch/jslack/api/methods/impl/MethodsClientImpl.java]: hmm, can you adjust your IDE settings not to expand wildcard imports? I'm using IntelliJ IDEA with default settings.

**Comment by joeltoby** [comment]
I've updated the PR ( and my IDE :) ).
Also just committed a small update to the readme

File: pom.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.github.seratch</groupId>
     <artifactId>jslack</artifactId>
-    <version>1.0.14</version>
+    <version>1.0.15</version>
```
Comment (seratch) [inline pom.xml]: You don't need to care this.

**Comment by seratch** [comment]
Thanks.

**Comment by anandmuthu-vnix** [comment]
@seratch Can you please make an jslack release with conversation API?

**Comment by seratch** [comment]
@anandmuthu-vnix I've released 1.0.15.

**Pull request merged**